src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
According to Enum [Convetion](https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html), these variables are constants, so the names of an enum type's fields should be in uppercase letters